Shaders/Readback/Filler.shader

```diff
@@ -0,0 +1,34 @@
+{
+    "Source" : "Filler.azsl",
+
+    "DepthStencilState" : { 
+        "Depth" : { 
+            "Enable" : false
```

Comment:
Is depth needed for this sample?

Reply:
Nope, it's really just rendering a uv pattern on a full screen triangle.